wegodang/products/views.py

```diff
@@ -1,3 +1,32 @@
-from django.shortcuts import render
+import jwt
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Q
+
+from products.models  import Category, Product, ProductModel, ProductImage
+
+class ProductView(View):
+    def get(self, request, product_id):
+        product = Product.objects.get(id = product_id)
+
+        product_images = product.product_images.all()
+
+        product_images = [{
+            "id"  : product_image.id,
+            "url" : product_image.image_url
+        } for product_image in product_images ]
+
+        products = {
+            "product_id"     : product.id,
+            "product_name"   : product.name,
+            "goal_percent"   : int(product.total_price / product.goal_price * 100),
+            "total_price"    : int(product.total_price),
+            "suppoters"      : product.suppoters,
+            "remaining_days" : (product.end_date - product.start_date.today()).days,
+            "story"          : product.story,
+            "image_url"      : product_images
+            }
```

Comment:
1. 마지막에 중가로 닫을 때의 들여쓰기 부분 수정해 주세요.
2. remaining_days 값이 잘 나오는지 궁급합니다. postman 통신 결과 첨부해 주세요.
```suggestion
        products = {
            "product_id"     : product.id,
            "product_name"   : product.name,
            "goal_percent"   : int(product.total_price / product.goal_price * 100),
            "total_price"    : int(product.total_price),
            "suppoters"      : product.suppoters,
            "remaining_days" : (product.end_date - product.start_date.today()).days,
            "story"          : product.story,
            "image_url"      : product_images
        }
```

Reply:
날짜를 계산하는 코드를 수정하였습니다 !

Reply:
<img width="985" alt="스크린샷 2022-07-11 오후 2 45 32" src="https://user-images.githubusercontent.com/104283159/178196794-9705d0be-3636-4261-a066-39c4ff45bfd9.png">